crates/terminal/src/terminal.rs

```diff
@@ -926,6 +942,20 @@ impl Terminal {
                 } else if let Some(url_match) = regex_match_at(term, point, &mut self.url_regex) {
                     let url = term.bounds_to_string(*url_match.start(), *url_match.end());
                     Some((url, true, url_match))
+                } else if let Some(python_match) =
+                    regex_match_at(term, point, &mut self.python_file_line_regex)
+                {
+                    let file_line =
+                        term.bounds_to_string(*python_match.start(), *python_match.end());
+
+                    let (file_path, line_number) = python_extract_path_and_line(file_line.as_str())
+                        .expect("Cannot parse python file line syntax");
```

Comment:
I understand that we did a python regex match and should not fail here, but the logic may have flaws (or gain, with later contributions) causing None to be returned here.

For the sake of less panics due to unwrap/expect's in "prod" code, let's replace it with `log_err()?` or, `debug_panic!` if you want to pay more attention to this in debug builds.

I also think we can avoid this entirely, and redo this `else if let` into `else`, then get the `let file_line` and use the regular Rust's regex (see how unit tests do that with `fn re_test` now) to match the rest of the cases, with capture values extractions?